########################################
# Messages
########################################
float32 ctr
float32 hdr
